submissions/ses3332002/dom/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
```

Comment:
It's better to place the `script` tag at the end of the `body` tag. It can help to load your page faster

Reply:
Well, this page is nearly empty :) and to give this page some content I need to JS load as fast as it can. So, I placed it to <head>.
But I can remake it if this is necessary.

Reply:
ready